test/functional/sync_plan/info_test.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/sync_plan'
+require_relative 'sync_plan_helpers'
+require_relative '../organization/organization_helpers'
+module HammerCLIKatello
+  describe SyncPlan::InfoCommand do
+    include OrganizationHelpers
+    include SyncPlanHelpers
+
+    it 'by ID' do
+      id = 1
+      api_expects(:sync_plans, :show) do |p|
+        p['id'] == id
+      end
+      arguments = "sync-plan info --id #{id}"
+      run_cmd(arguments.split)
+    end
+
+    it 'by organization id and sync plan name' do
```

Comment:
This should be two tests: one for testing sync plan name with organization ID and one that tests that organization options are required when providing sync plan name.